docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -496,30 +496,110 @@ A cicd_task is a single job of ci/cd.
+| `security_category`         | varchar  | 100        | The security category of the issue                                      |         |
+| `creation_date`             | datetime | 3          | The date and time when the issue was created                            |         |
+| `update_date`               | datetime | 3          | The date and time when the issue was last updated                       |         |
+
+#### cq_issue_code_blocks
+
+| **field**      | **type** | **length** | **description**                                                                       | **key** |
+| :------------- | :------- | :--------- | :------------------------------------------------------------------------------------ | :------ |
+| `id`           | varchar  | 255        | This key is generated based on details from the original plugin                       | PK      |
+| `issue_key`    | varchar  | 255        | A string that stores the key of the issue that the code block is associated with      |         |
+| `component`    | varchar  | 255        | A string that stores the name of the component that the code block is associated with |         |
+| `start_line`   | bigint   | 255        | An integer that stores the line number where the code block starts                    |         |
+| `end_line`     | bigint   | 255        | An integer that stores the line number where the code block ends                      |         |
+| `start_offset` | bigint   | 255        | An integer that stores the line number where the code block ends                      |         |
+| `end_offset`   | bigint   | 255        | An integer that stores the offset where the code block ends                           |         |
+| `msg`          | longtext |            | A long text field that stores the message associated with the code block              |         |
+
+#### cq_file_metrics
+
+| **field**                    | **type** | **length** | **description**                                                  | **key** |
+| :--------------------------- | :------- | :--------- | :--------------------------------------------------------------- | :------ |
+| `id`                         | varchar  | 255        | This key is generated based on details from the original plugin  | PK      |
+| `project_key`                | varchar  | 255        | The key of the project that the issue belongs to                 | PK      |
+| `file_name`                  | longtext |            | longtext fields that store the name of the file                  |         |
+| `file_path`                  | longtext |            | longtext fields that store the path of the file                  |         |
+| `file_language`              | longtext |            | longtext fields that store the language of the file              |         |
+| `code_smells`                | bigint   |            | numeric fields that store various metrics of the file            |         |

```

Comment:
1. The description can be updated to: Code smells of this file.
2. Same goes to the description below